spec/extensions.spec.js

```diff
@@ -0,0 +1,10 @@
+describe('Number.prototype.toFixedIfLarger', () => {
+  it('true', () => {
+    expect(true);
+  });
+  // TODO - figure out why toFixedIfLarger is not in scope
```

Comment:
you have to import file patching prototype. But lets skip prototypes all together